packages/spindle-ui/src/Modal/AppealModal.css

```
@@ -0,0 +1,212 @@
+/*
+ * AppealModal
+ * NOTE: Styles can be overridden with "--AppealModal-*" variables
+*/
+:root {
+  --AppealModal-image-border: 1px inset var(--color-border-low-emphasis);
+  --AppealModal-image-borderRadius: 8px;
+}
+
+.spui-AppealModal {
+  background: var(--color-surface-primary);
+  border: none;
+  border-radius: 20px;
+  /* TODO: replace color with color palette */
+  box-shadow: 0 11px 28px rgba(8, 18, 26, 0.12);
+  padding: 0;
+  width: 400px;
+}
+
+/* stylelint-disable plugin/selector-bem-pattern */
+html:has(.spui-AppealModal[open]) {
+  overflow: hidden;
+}
+
+/* Fallback for :has() */
+html.spui-AppealModal--open {
+  overflow: hidden;
+}
+/* stylelint-enable plugin/selector-bem-pattern */
+
+/* The minimum margin for the small devices */
+/* 440px is calculated from dialog width (400px) + minimum margin (20px * 2) */
+/* width property is calculated from 100% - (minimum margin * 2) */
+@media screen and (max-width: 440px) {
+  .spui-AppealModal {
+    width: calc(100% - (20px * 2));
+  }
+}
+
+.spui-AppealModal::backdrop {
+  /* TODO: replace color with color palette, backdrop does not accept variable */
+  background: rgba(0, 0, 0, 0.6);
+}
+
+/* Dialog polyfill adds ".backdrop" instead of "::backdrop" */
+/* stylelint-disable-next-line plugin/selector-bem-pattern */
+.spui-AppealModal + .backdrop {
+  background: rgba(0, 0, 0, 0.6);
+}
+
+/*
+ * Override focus styles in Dialog. This can be removed when Safari supports the following styles.
+ * - Safari displays focus ring even when :focus-visible
+ * - Safari does not support radius outline
+*/
+.spui-AppealModal
+  :is(.spui-Button, .spui-LinkButton, .spui-IconButton, .spui-SubtleButton):focus {
+  box-shadow: 0 0 0 1px var(--color-surface-secondary),
+    0 0 0 3px var(--color-focus-clarity);
+  outline: none;
+}
+
+.spui-AppealModal
+  :is(.spui-Button, .spui-LinkButton, .spui-IconButton, .spui-SubtleButton):focus:not(:focus-visible) {
+  box-shadow: none;
+}
+
+.spui-AppealModal-frame {
+  color: var(--color-text-medium-emphasis);
+  display: grid;
+  gap: 0px 0px;
```

```suggestion
  gap: 0 0;
```

gapって `0` に省略できないんでしたっけ？

---

全然いけますね、ありがとうございます！